internal/tools/bigquery/bigqueryforecast/bigqueryforecast.go

```diff
@@ -50,6 +51,8 @@ type compatibleSource interface {
 	BigQueryRestService() *bigqueryrestapi.Service
 	BigQueryClientCreator() bigqueryds.BigqueryClientCreator
 	UseClientAuthorization() bool
+	IsDatasetAllowed(projectID, datasetID string) bool
+	BigQueryAllowedDatasets() []string
```

Comment:
```suggestion
	AllowedDatasets() []string
```

Keep consistency with the other PRs. I think just `AllowedDatasets()` should be good to keep it concise.

Reply:
This returns s.AllowedDatasets in a slightly different format, so bigquery added to make the function and variable name different.